### PR TITLE
chore(compass): remove global tag selector styles from bootstrap, update styles accordingly

### PR DESCRIPTION
**Pull request by Anemy**
Part of [COMPASS-5579](https://jira.mongodb.org/browse/COMPASS-5579)

This PR removes most of the global tag styling selecting from bootstrap. These styles are currently namespaced so they only impact the connected views of Compass. The other changes in this PR update styles in places where we were depending on the globals. It's tough to find everything these styles touch in Compass we should definitely keep an eye out for anything looking off. I've run through every view I can think of in Compass a few times but I can definitely miss things.

### CODE REVIEW

File: packages/compass-aggregations/src/components/settings/settings.jsx [hunk 1]
```diff
@@ -76,11 +77,11 @@ class Settings extends PureComponent {
       return (
         <div className={classnames(styles['input-group'])}>
           <div className={classnames(styles['input-meta'])}>
-            <label htmlFor='aggregation-limit'>Limit</label>
-            <p id="aggregation-limit-description">
+            <Label htmlFor='aggregation-limit'>Limit</Label>
+            <Description id="aggregation-limit-description">
```
Comment (Anemy) [inline packages/compass-aggregations/src/components/settings/settings.jsx:81]: Before/After of this page:

Screen Shot 2022-03-21 at 3 16 51 PM
Screen Shot 2022-03-21 at 3 16 39 PM

**Comment by Anemy** [comment]
Looks like tests just timed out, but not seeing a failure, I think it's a flake.

**Review by mabaasit — APPROVED** [review]

**Comment by Anemy** [comment]
Not sure why the e2e tests are failing on github, it's passing on local + evergreen. Verifying if it's flake before merging

**Comment by Anemy** [comment]
Yeah seeing it on main in previous commits, merging

**Pull request merged**